src/screens/ProfileScreen/components/ProfileCard.tsx

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+import {
+  ProfileCardContainer,
+  Avatar,
+  Name,
+  Email,
+  RoleBadge,
+  RoleText,
+  SpecialtyText,
+} from "../styles";
+import { getRoleText } from "../services/roleService";
+import { User } from "../models/User";
+
+type Props = {
+  user: User | null;
```

Comment:
```suggestion
  user: User;
```
Sem opcional aqui, não faz sentido esse componente sequer existir sem um User